crates/orderbook/src/api/order_validation.rs

```diff
@@ -679,24 +681,33 @@ mod tests {
 
     #[tokio::test]
     async fn pre_validate_ok() {
+        let liquidity_order_owner = H160::from_low_u64_be(1);
         let min_order_validity_period = Duration::from_secs(1);
         let validator = OrderValidator::new(
             Box::new(MockCodeFetching::new()),
             dummy_contract!(WETH9, [0xef; 20]),
-            vec![],
-            Duration::from_secs(1),
+            hashset!(),
+            hashset!(liquidity_order_owner),
+            min_order_validity_period,
             Arc::new(MockMinFeeCalculating::new()),
             Arc::new(MockBadTokenDetecting::new()),
             Arc::new(MockBalanceFetching::new()),
         );
+        let order = || PreOrderData {
+            valid_to: shared::time::now_in_epoch_seconds()
+                + min_order_validity_period.as_secs() as u32
+                + 2,
+            sell_token: H160::from_low_u64_be(1),
```

Comment:
nit: Little surprising that the `sell_token` uses the same address as `liquidity_order_owner`.

Reply:
Good point, it doesn't matter that they are the same, but it makes the intent of the test less clear. Will change.

Reply:
Fixed.